tests/UNMOCKED_end2end_tests/real_active_learning_run.py

```diff
@@ -0,0 +1,121 @@
+    else:
+        print("Skip upload of dataset: already uploaded.")
+
+    # 2. upload the embeddings of the dataset
+    api_workflow_client.upload_embeddings(path_to_embeddings_csv=path_to_train_embeddings_csv, name=f"embedding_1")
+
+    # define the active learning agent and dataset
+    agent = ActiveLearningAgent(api_workflow_client)
+    training_set = CSVEmbeddingDataset(path_to_embeddings_csv=path_to_train_embeddings_csv)
+    test_set = CSVEmbeddingDataset(path_to_embeddings_csv=path_to_test_embeddings_csv)
+    no_samples_total = len(training_set.dataset.items())
+
+    al_scorer = None
+
+    for iteration, ratio in enumerate(ratios):
+        print(f"Beginning with iteration {iteration}")
+        # 3. Perform a sampling
+        n_samples = int(ratio * no_samples_total)
+        method_here = SamplingMethod.CORESET if iteration == 0 and method == SamplingMethod.CORAL else method
+        sampler_config = SamplerConfig(method=method_here, n_samples=n_samples)
+        if al_scorer is None:
+            agent.query(sampler_config=sampler_config)
+        else:
+            agent.query(sampler_config=sampler_config,al_scorer=al_scorer)
+
+        # 4. get the features and labels of the labeled_set
+        features = training_set.get_features(agent.labeled_set)
+        labels = training_set.get_labels(agent.labeled_set)
+
+        # 5.1 train a classifier
+        classifier = KNeighborsClassifier(n_neighbors=3)
+        classifier.fit(features, labels)
+        # 5.2 compute the new accuracy
+        test_features, test_labels = test_set.all_features_labels()
+        accuracy = classifier.score(test_features, test_labels)
+        print(f"Accuracy of classifier: {accuracy:.3}")
+
+        if iteration != len(ratios)-1:  # skip the following for the last iteration
+            # 5.3 predict on the unlabeled set
+            predictions = classifier.predict_proba(training_set.get_features(agent.unlabeled_set))
+
+            # 6. Save the predictions in a scorer
+            al_scorer = ScorerClassification(model_output=predictions)
+
+
+if __name__ == "__main__":
+    path_to_dataset = "/Users/malteebnerlightly/Documents/datasets/clothing-dataset-small-master/train"
+    token = os.getenv("TOKEN")
+    dataset_id = "603606ed2176760032373065"
+    path_to_train_embeddings_csv = "/Users/malteebnerlightly/Documents/datasets/clothing-dataset-small-master/train/lightly_outputs/2021-02-23/23-38-25/embeddings.csv"
+    path_to_test_embeddings_csv = "/Users/malteebnerlightly/Documents/datasets/clothing-dataset-small-master/test/lightly_outputs/2021-02-23/23-41-09/embeddings.csv"
+    for i in range(1):
+        print(f"ITERATION {i}:")
+        real_active_learning_run(path_to_dataset, token, dataset_id,
+                                 path_to_train_embeddings_csv=path_to_train_embeddings_csv,
+                                 path_to_test_embeddings_csv=path_to_test_embeddings_csv)
+        print("")
```

Comment:
Since the whole script is a "test" you should also mention what the expected output looks like. Print "success" if it runs through or compare values or something that tells the user "it worked, test passed"